src/cloudant/document.py

```diff
@@ -63,7 +63,8 @@ def __init__(self, database, document_id=None):
         self._document_id = document_id
         if self._document_id is not None:
             self['_id'] = self._document_id
-        self.encoder = self._client.encoder
+        self.decoder = kwargs.get('decoder') or self._client.decoder
```

Comment:
If the decoder is not configurable, can we remove `kwargs.get('decoder')`?

Reply:
It's only not configurable from the Cloudant client. We are allowing users to specify a custom decoder in the Document constructor.

A Client level decoder will be added in a future PR. It requires a little more work to ensure the decoder is passed around correctly.